app/helpers/view_components/student/tracks_list.rb

```diff
@@ -20,6 +21,18 @@ def to_s
       ].freeze
       private_constant :STATUS_OPTIONS
 
+      TAG_OPTIONS = [
```

Comment:
Can you help me with this?

1. Is the structure okay?
2. Is the naming okay?

Reply:
Yeah - this is totally fine for now. We'll probably change it once we've thought it through further, but this works for now 🙂